exercises.ipynb

```diff
@@ -81,15 +94,17 @@
     "    # titles from the BeautifulSoup object.\n",
     "    #\n",
     "    # Return the titles\n",
-    "    pass"
+    "    raw_titles = requests.get(url)\n",
+    "    raw_titles2 = BeautifulSoup(raw_titles.content, 'html.parser')\n",
```

Comment:
I really dislike variables named with "2" -- There's no way that helps anyone reading! If you don't need the previous variable, you can just replace it, that's fine! Otherwise, come up with a better name ;)